packages/adapter/test/fixtures/ois.ts

```diff
@@ -102,6 +99,10 @@ export function buildOIS(overrides?: Partial<OIS>): OIS {
         ],
       },
     ],
+    credentials: {
+      securityScheme: 'myapiApiScheme',
+      value: 'value',
```

Comment:
Nit: I think we should try to use more descriptive values in fixtures where we can:
```suggestion
      value: 'supersecret',
```